torch/ao/quantization/_pt2e/graph_utils.py

```diff
@@ -20,15 +20,23 @@
 ]
 
 
-def _create_equivalent_types_dict():
+def get_equivalent_types() -> List[Set]:
+    return _EQUIVALENT_TYPES
+
+def _create_equivalent_types_dict(equivalent_types=None):
```

Comment:
You dont really need to override `_create_equivalent_types_dict`.

Reply:
Thanks for the comments and remove it.